src/components/CopyButton.vue

```diff
@@ -6,11 +6,11 @@
     :data-clipboard-target="el"
   >
     <span v-if="!success">
-      <i class="icon cc-share me-2" />
+      <i class="icon cc-share mx-1" />
```

Comment:
The [transitional styles](https://www.figma.com/file/BlUTIXRHLOq5mRITpDpi3E/Transitional-styles?node-id=530%3A2051) don't have an icon on the copy button, so I think we can remove the icons altogether.

Reply:
Agreed, we should remove the icon. CC @panchovm

Reply:
Good call, thanks. I should have checked the mockups before, It seemed strange to me that the title was so small, and of course, it's fixed in the transitional mockups :) I'll make that change too.